src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java

```diff
@@ -235,127 +301,115 @@ public void testWrapAssertionErrorWithoutMessage()
         }
     }
 
+    @Test()
+    public void testIsThrowableCausedBy()
```

Comment:
add `@SuppressBrowsers` since a browser is not needed for this unit test

Reply:
Removed